devcluster/__main__.py

```diff
@@ -52,6 +69,34 @@ def get_host_addr_for_docker() -> Optional[str]:
     return None
 
 
+def maybe_install_default_config():
+    # Don't bother asking the user if the user isn't able to tell us.
+    if not sys.stdin.isatty():
+        return None
+
+    # Decide where we would install the config file.
+    if "XDG_CONFIG_HOME" in os.environ:
```

Comment:
use `appdirs.user_config_dir`?
https://pypi.org/project/appdirs/

in the main repo, `det deploy` already does (and there's one place in `determined/common/util.py` which I need to refactor to also use it)

Reply:
Yeah, have another look.

Reply:
🤣